Source/WTF/wtf/SystemTracing.h

```diff
@@ -163,10 +163,21 @@ enum TracePointCode {
     WebXRCPFrameStartSubmissionEnd,
     WebXRCPFrameEndSubmissionStart,
     WebXRCPFrameEndSubmissionEnd,
+    DisplayLinkUpdate,
```

Comment:
Right above this enum, there's a comment that reads:

> // When adding or changing these codes, update Tools/Tracing/SystemTracePoints.plist to match.

This file seems to still exist, so I think this should be done.

FWIW, the reason I suggested not using this enum / TraceScope was specifically to avoid editing the plist file. I can see how TraceScope is massively useful though!

Maybe it's worth double checking with Apple folks if just editing the plist file is enough.

Reply:
Forgot about the plist file, thanks for mentioning. The only thing that needs to be added there would be 'DisplayLinkUpdate', since the other new tracing points are GTK/WPE specific.

I remember from the past: we shall not add enum entries in the middle, but append.
I do want to double-check with Apple. @smfr, maybe?

Reply:
In fact ` Tools/Tracing/SystemTracePoints.plist` is probably the reason why we cannot change enum values "in the middle", but appending works. I've added a new entry for 'DisplayLinkUpdate', but not for the GTK/WPE specific ones.